ui/src/sources/containers/ManageSources.js

```diff
@@ -124,4 +121,17 @@ export const ManageSources = React.createClass({
   },
 });
 
-export default withRouter(ManageSources);
+function mapStateToProps(state) {
+  return {
+    sources: state.sources,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    loadSourcesAction: bindActionCreators(loadSourcesAction, dispatch),
+    removeSourceAction: bindActionCreators(removeSourceAction, dispatch),
```

Comment:
We haven't set a good example on this, but more often than not, mapDispatchToProps doesn't need to be a function. `connect` accepts a function or an object. so you can just pass `{ loadSourcesAction, removeSourceAction }` to `connect` and that will achieve the same effect.